frontend/src/app/(guest)/accommodations/[id]/page.tsx

```diff
@@ -34,7 +35,8 @@ const GuestAccommodationsDetails: React.FC<
   //  check if listable type of listing received is an experience or somewhat undefined to avoid passing experience in accommodation details page
   if (
     accData.listable_type.split("\\")[2] === "Experience" ||
```

Comment:
I have already created a getListingType function in utlis/helpers, maybe it can be used here